src/variables/variable_binary_gas_connection_flow.jl

```diff
@@ -47,7 +47,7 @@ function binary_gas_connection_flow_indices(
                 stochastic_scenario=stochastic_scenario,
                 t=t,
                 temporal_block=temporal_block)
-        if connection_binary_gas_flow(connection=conn) == true && has_state(node=n) == false
+        if connection_binary_gas_flow(connection=conn)  && !has_state(node=n)

```

Comment:
What about renaming `connection_binary_gas_flow` to `has_binary_gas_flow`  or something like that? It seems that we use `has_{something}` for this type of boolean values?